src/components/Team/TeamCard/index.tsx

```diff
@@ -17,7 +17,7 @@ export default function TeamCard({ name, position, src, alt }: TeamMember) {
         />
       </div>
       <div className="py-5">
-        <h3 className="text-xl font-bold text-red-400 capitalize relative mb-5">
+        <h3 className="text-xl font-bold text-red-400 capitalize mb-5">
```

Comment:
The `position: relative` is important here as that allows for the line next to the dot to show up. To fix the bug look into the `z-index` attribute

Previous: (On Dev Site)
![image](https://github.com/theDS3/Website/assets/32755895/dd445c63-b261-4eba-bedc-9b26101aab93)

Current:
![image](https://github.com/theDS3/Website/assets/32755895/11e482c0-8de1-4fa8-9a4c-ba413723eddb)